alternative_backend/alternative_backend/settings.py

```diff
@@ -40,7 +40,8 @@
     'rest_framework',
     'workers',
     'boards',
-    'stations'
+    'stations',
+    'orders'
```

Comment:
W miare możliwości zawsze stawiaj przecinek po ostatnim elemencie.
Wtedy jak dodajesz kolejny elemnet to GIT pokazuje zmiane w 1 wierszu, a nie w 2.
Zmiana w lini 43 to tylko dodanie przecinka na końcu, można było tego uniknąć i nie zaśmiecać (łatwiej potem robić review)